docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -126,16 +131,43 @@ To add external data to the knowledge base in {kib}:
 ----
 {
   "id": "a_unique_human_readable_id",
-  "text": "Contents of item",
+  "text": "Contents of item"
 }
 ----
 
+(*TBD* Add screenshot of how the UI looks?)
```

Comment:
You want to be very intentional about adding screenshots—updating them is very time-consuming. Our products change frequently, meaning screenshots go out of date quickly. We don't have a good rule in our style guide about this yet (I brought this up in #style-guide writing).

Reply:
Completely agreed. As talked we will try to not rely on screenshots and for example we will add a small image of the AI Assistant icon instead of using a screenshot for that purpose.